vcd/datasource_vcd_resource_list_test.go

```diff
@@ -22,60 +22,133 @@ type listDef struct {
 
 func TestAccVcdDatasourceResourceList(t *testing.T) {
 	preTestChecks(t)
+	// TODO add checks
```

Comment:
Could you be more explicit about what checks should be added - not having context I already don't know - most probably the "not empty" checks...

Reply:
removed as checks were added